src/tink/http/Container.hx

```diff
@@ -31,7 +31,7 @@ class NodeContainer implements Container {
       application.serve(
         new IncomingRequest(
           req.socket.remoteAddress, 
-          new IncomingRequestHeader(cast req.method, req.url, req.httpVersion, [for (name in req.headers.keys()) new HeaderField(name, (req.headers[name]: String))]), 
+          new IncomingRequestHeader(cast req.method, req.url, req.httpVersion, [for (name in req.headers.keys()) for(value in (req.headers[name]:String).split(', ')) new HeaderField(name, value)]), 
```

Comment:
Unfortunately, it's not as simple as that. The distinction the nodejs doc makes for different headers is crucial.

Consider this request header (taken from the W3C's spec): `If-Modified-Since: Sat, 29 Oct 1994 19:43:31 GMT`. Notice the occurrence of the substring `", "` in it and how it would lead to the header being split.

Headers are a mess ... trying to fix that is no small task ;)